lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -684,6 +677,33 @@ HdVP2Material::HdVP2Material(HdVP2RenderDelegate* renderDelegate, const SdfPath&
     : HdMaterial(id)
     , _renderDelegate(renderDelegate)
 {
+#ifdef WANT_MATERIALX_BUILD
```

Comment:
This looks like one time initialization. If it is we can wrap it in a std::once to make sure it only happens on time.

Reply:
No. Definitely one-time. Will also move it to the C++ file as requested.